arangod/GeneralServer/RestHandler.cpp

```diff
@@ -95,9 +97,9 @@ void RestHandler::setStatistics(RequestStatistics* stat) {
   }
 }
 
-bool RestHandler::forwardRequest() {
+futures::Future<Result> RestHandler::forwardRequest(bool& forwarded) {
   if (!ServerState::instance()->isCoordinator()) {
```

Comment:
though the caller sets `forwarded` to false before calling this method, I think it would be better if the very first line in this method here set `forwarded` to false. That way it is still safe if the caller forgets to set the flag before calling.